content/applications/hr/payroll/reporting.rst

```diff
@@ -0,0 +1,205 @@
+=========
+Reporting
+=========
+
+The :guilabel:`Reporting` section offers a variety of reports to choose from. The
+:guilabel:`Payroll`, :guilabel:`Meal Vouchers`, :guilabel:`Attachment of Salary`, and
+:guilabel:`DMFA` reports are universal and available for all companies regardless of location, while
+the rest of the reports are specific to Belgium only.
```

Comment:
Not sure if it's necessary, but it might be good to mention the Belgium-specific reports by name.